data/kamailio.cfg

```diff
@@ -1267,10 +1552,14 @@ route[SEND_ES_STATS] {
 	$http_req(method) = "POST";
 	$http_req(hdr) = "Content-Type: application/json; charset=UTF-8";
 
+	#!ifdef TO_GRAYLOG
+	$var(graylogurl) = "http://YOUR_GRAYLOG_IP:12201/gelf";
+	http_async_query("$var(graylogurl)", "$var(graylog_json)", "ES_REPLY");
+	#else
 	# Index+Type defined in bulk object
-	$var(elasticurl) = "ELASTICSEARCH_URL/_bulk";
-
+	$var(elasticurl) = "http://YOUR_ELASTIC_IP:9200/_bulk";
```

Comment:
In this way you are forcing the protocol and the port on which elasticsearch (or garylog) run. I still think here is better to have the full url which could be composed elsewhere by configuration variables (e.g.: transport, ip, port).